package.json

```diff
@@ -115,41 +115,42 @@
     "react-native-android-location-services-dialog-box": "^2.8.2",
     "react-native-animatable": "^1.3.3",
     "react-native-ble-plx": "2.0.1",
-    "react-native-camera": "3.19.0",
-    "react-native-config": "0.11.7",
+    "react-native-camera": "3.43.6",
+    "react-native-config": "1.4.2",
     "react-native-crypto": "^2.2.0",
     "react-native-easy-markdown": "^2.0.0",
     "react-native-extra-dimensions-android": "^1.2.5",
-    "react-native-fingerprint-scanner": "^4.0.0",
-    "react-native-gesture-handler": "^1.6.0",
-    "react-native-keychain": "^5.0.0",
+    "react-native-fingerprint-scanner": "^6.0.0",
+    "react-native-gesture-handler": "^1.10.3",
+    "react-native-keychain": "^7.0.0",
     "react-native-level-fs": "^3.0.0",
     "react-native-locale": "0.0.19",
-    "react-native-modal": "^11.5.4",
+    "react-native-modal": "^11.10.0",
     "react-native-navigation-bar-color": "^2.0.1",
     "react-native-os": "^1.2.6",
-    "react-native-progress": "^4.0.3",
-    "react-native-qrcode-svg": "6.0.3",
-    "react-native-randombytes": "^3.6.0",
-    "react-native-reanimated": "^1.10.1",
-    "react-native-redash": "^15.11.1",
-    "react-native-safe-area-context": "^0.7.3",
+    "react-native-pager-view": "^5.1.5",
+    "react-native-progress": "^4.1.2",
+    "react-native-qrcode-svg": "6.1.1",
+    "react-native-randombytes": "^3.6.1",
+    "react-native-reanimated": "^2.1.0",
+    "react-native-redash": "^16.0.11",
+    "react-native-safe-area-context": "^3.2.0",
     "react-native-safe-area-view": "^1.1.1",
-    "react-native-screens": "^2.15.0",
+    "react-native-screens": "^3.1.1",
     "react-native-sentry": "0.43.2",
     "react-native-share": "^5.0.0",
```

Comment:
Skip upgrading this time due to https://github.com/react-native-share/react-native-share/issues/913#issuecomment-813446599